easysnmp/interface.c

```diff
@@ -1517,7 +1546,9 @@ static PyObject *create_session_capsule(SnmpSession *session)
     ctx->handle = handle;
     ctx->invalid_oids = (bitarray *) ctx->invalid_oids_buf;
     bitarray_buf_init(ctx->invalid_oids, sizeof(ctx->invalid_oids_buf));
-    return (capsule);
+
+    return capsule;
```

Comment:
I generally discourage unnecessary churn like this, but this seems to be making the style of the line of code more in line with the general style.

Reply:
done, thanks